custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -396,6 +396,4 @@ def indexes(self):
         return [
             'CREATE INDEX IF NOT EXISTS chm_case_idx ON "{}" (case_id)'.format(self.tablename),
             'CREATE INDEX IF NOT EXISTS chm_awc_idx ON "{}" (awc_id)'.format(self.tablename),
-            'CREATE INDEX IF NOT EXISTS chm_mother_dob ON "{}" (mother_case_id, dob)'.format(self.tablename),
-            'CREATE INDEX IF NOT EXISTS chm_month_supervisor_id ON "{}" (month, supervisor_id)'.format(self.tablename),
```

Comment:
This one was added fairly recently in https://github.com/dimagi/commcare-hq/pull/25477. However we now pull agg_child_health from the staging table which means that we are only pulling this months data. Because of this the index is now useless.

Reply:
Actually before removing this, maybe we should have the MBT code pull from the staging table?

Also realizing that this may affect ad-hoc queries if they need to filter child_health_monthly by month. Let me know your thoughts